src/main/java/use_cases/Import.java

```diff
@@ -0,0 +1,36 @@
+package use_cases;
+
+import java.util.List;
+import entities.Item;
+import entities.TempDataStorage;
+import interface_adaptors.ImportPresenter;
+
+public class Import implements ImportInputBoundary{
+    private final ImportOutputBoundary presenter;
+    public Import(ImportOutputBoundary presenter){
+        this.presenter = presenter;
+    }
+    public String importDatabase(ImportDS importData){
+        List<String[]> data = importData.getImportData();
+        data.remove(0);
```

Comment:
I assume this means removing the headline in our data file which does not contain any actual data.